packages/client/src/navigation/routes.ts

```diff
@@ -96,3 +96,5 @@ export const USER_PROFILE = '/userProfile/:userId'
 export const FORM_CONFIG_WIZARD = '/config/form/wizard/:event/:section'
 
 export const VIEW_RECORD = '/:declarationId/viewRecord'
+// export const LIST_OF_ORGANIZATIONS = '/organization/'
```

Comment:
Let's remove all commented-out code from the PR

Reply:
done